force-di/test/classes/di_BindingParamTest.cls

```diff
@@ -44,19 +44,19 @@ private class di_BindingParamTest {
 
         response = di_BindingParam.invoke( new di_BindingParam.Request[] { request } )[0];
 
-        System.assertEquals( null, response.longValue );
-        System.assertEquals( null, response.decimalValue );
-        System.assertEquals( null, response.stringValue );
-        System.assertEquals( null, response.booleanValue );
-        System.assertEquals( null, response.dateValue );
-        System.assertEquals( null, response.dateTimeValue );
-
-        System.assertEquals( 0, response.longValues.size() );
-        System.assertEquals( 0, response.decimalValues.size() );
-        System.assertEquals( 0, response.stringValues.size() );
-        System.assertEquals( 0, response.booleanValues.size() );
-        System.assertEquals( 0, response.dateValues.size() );
-        System.assertEquals( 0, response.dateTimeValues.size() );
+        System.Assert.isNull(response.longValue );
```

Comment:
@chazwatkins -- @daveespo and I were discussing these changes.  One thing that Dave noticed was that the `isNull` method's error message actually doesn't express what he condition value is if it is not null.  The older `assertEquals` method error message does express what the value is.

We would like a change, please, in these instances where the no parameter `isNull` method is used to instead be the `System.Assert.areEqual(null, Object)` method.  

Please let me know if you have questions.

Reply:
Sure, I'll make the updates on all the repos